src/Playwright/WebSocket.cs

```diff
@@ -48,78 +48,5 @@ internal WebSocket(IChannelOwner parent, string guid, WebSocketInitializer initi
         public string Url => _initializer.Url;
 
         public bool IsClosed { get; internal set; }
-
-        public async Task<T> WaitForEventAsync<T>(PlaywrightEvent<T> webSocketEvent, Func<Task> action = default, Func<T, bool> predicate = default, float? timeout = default)
```

Comment:
What happened to websocket? Do we not produce RunAndWaitForEvent here?

Reply:
I nuked it from web socket for now, only listeners there.